CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased]
 ------------
 
+- `oe-gdb` renamed to `oegdb` for consistency with other tools, such as
```

Comment:
Please put this under the `### Changes` header.

Also, we never defined a consistent writing style for the changelog. I've been going by the commit message guidelines (Lead with active imperative verb), we should probably clarify that.